### PR TITLE
Add file with information to create boot files to build Chez

### PR DESCRIPTION
**Pull request by pmatos**

### CODE REVIEW

**Comment by pmatos** [comment]
So this PR is to merge this doc file into the riscv port branch.
We can either work with both ports off master, both ports off the same branch or with a port per branch.

I know I have `wip-riscv` already but for you to better understand the code I thought about splitting it into PRs that you can review separately and possibly create similar PRs for the aarch64 port. What do you think?

File: BOOTFILE_RACKET_BUILD.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+* Install the package `cs-bootstrap` (`-D` will skip building documentation).
+```
+$ raco pkg install -i -D cs-bootstrap
+```
+
+* Setup the boot files architecture. For `i386` (32-bits intel) use `MACH=i3le` (without threads) or `MACH=i3le` (with threads). For `x86_64` (64-bits intel) use `MACH=a6le` (without threads) or `MACH=ta6le` (with threads).
+```
+$ cd ChezScheme
+ChezScheme $ MACH=... racket -l cs-bootstrap
```
Comment (jessealama) [inline BOOTFILE_RACKET_BUILD.md]: What is that extra `$` there after `ChezScheme`?  Is that really intended? It looks like a typo to me.

File: BOOTFILE_RACKET_BUILD.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+```
+
+Should finish by printing:
```
Comment (jessealama) [inline BOOTFILE_RACKET_BUILD.md]: "This should finish..."

File: BOOTFILE_RACKET_BUILD.md (hunk 1 above)
Comment (pmatos) [reply BOOTFILE_RACKET_BUILD.md]: Part of the shell prompt, just to say that we are now inside ChezScheme directory. :) I can remove it.

**Comment by pmatos** [comment]
Closing this to reopen the PR against the new `ports` branch.

**Pull request closed without merging**